File: Treinamento/Java/Estrutura_sequencial/Questão10.java
import java.util.Scanner;
public class Questão10 {
    public static void main(String[] args) {
        Scanner s;
        s = new Scanner(System.in);

        System.out.print("Digite o peso de peixes pescados: ");
        double p = s.nextDouble();

        double e = p - 50;

        System.out.printf("Excesso de peixes: %.2f\n", e);
        System.out.printf("Multa: R$%.2f reais\n", e * 4);
    }
}

        

File: Treinamento/C/Estrutura_sequencial/Questão10.c
#include <stdio.h>

int main() {
    double peso;

    printf("Digite  a quiantidade de peixes pescada: ");
    scanf("%lf", &peso);

    if (peso > 50) {
        double excesso = peso - 50;
        printf("Multa a ser paga: R$%lf", excesso * 4.0);
    } else {
        printf("Quantidade de peixes regular");
    }
}
